### ISSUE TITLE
move new JWPlatform iframe to production

### ISSUE THREAD

**Issue by abrahammartin** [uisautomation/sms-webapp#37]
This task is dependant from stats (already done) but also from #34 #35 #36 #42 and #43 . Once all of this is done, change current SMS downloads servers to point apache to a redirect to the new UMS webapp.

**Comment by abrahammartin** [comment]
This is in #40 now
